spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/trace/GcpTraceProperties.java

```diff
@@ -86,6 +86,11 @@
 	 */
 	private int messageTimeout = 1;
 
+	/**
+	 * Timeout in seconds for server response. Ref: https://github.com/spring-cloud/spring-cloud-gcp/issues/1617.
+	 */
+	private Long serverResponseTimeoutMs = 5000L;
```

Comment:
Can we just leave it `null` by default so that we inherit the default from `Sender`?
It will be the same default.

Reply:
Changing to null breaks a load of tests (NPE due to the null). So leaving as 5000L for now. Assessing a sane default.

Reply:
From StackDriverSender the default is set to zipkin2.reporter.stackdriver.internal.UnaryClientCall.DEFAULT_SERVER_TIMEOUT_MS (5000L). So proposing to set the default to this constant to avoid NPE in setting up the default sender bean. A value of zero will also fail since builder validates for a timeout greater than zero.

Reply:
@meltsufin This does require an import from zipkin2 package What do you think?

Reply:
Can you just skip setting the value on `StackdriverSender.Builder` if the property is `null`?

Reply:
+1 .Done